pkg/resources/procedure_java.go

```diff
@@ -62,8 +62,12 @@ func CreateContextProcedureJava(ctx context.Context, d *schema.ResourceData, met
 	}
 	handler := d.Get("handler").(string)
 	runtimeVersion := d.Get("runtime_version").(string)
-	// TODO [this PR]: handle real packages
-	packages := []sdk.ProcedurePackageRequest{*sdk.NewProcedurePackageRequest("com.snowflake:snowpark:1.14.0")}
+
+	packages, err := parseProceduresPackagesCommon(d)
+	if err != nil {
+		return diag.FromErr(err)
+	}
+	packages = append(packages, *sdk.NewProcedurePackageRequest(fmt.Sprintf(`%s%s`, sdk.JavaSnowparkPackageString, d.Get("snowpark_package").(string))))
```

Comment:
from this I'm guessing snowpark doesn't have to be at first position, but has to be included?

Reply:
it seems so, I will check in other resources too